tests/end2end/features/utilcmds.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Miscellaneous utility commands exposed to the user.
+
+    Background:
+        Given I open data/scroll/simple.html
+        And I run :tab-only
+
+    ## :later
+
+    Scenario: :later before
+        When I run :later 500 scroll down
+        Then the page should not be scrolled
```

Comment:
Hmm, this is a bit problematic as it'll influence the next test, but as long as we don't have some way to cancel queued `:later` commands I see no way around that.